stages/org.osbuild.users

```diff
@@ -158,6 +169,10 @@ def main(tree, options):
                 print(f"Error: can't set uid of existing user '{name}'")
                 return 1
             usermod(tree, name, gid, groups, description, home, shell, password)
+
+            # ensure the home directory exists, see module doc string for details
+            _, _, _, _, _, home, _ = getpwnam(tree, name)
```

Comment:
`home = getpwnam(tree, name)[5]`?

Reply:
Ha, I had this first. Maybe we should really return a named tuple instead.